### ISSUE TITLE
Provide public API for infobar usage

### ISSUE THREAD

**Issue by Therzok** [mono/monodevelop#5876]
```
	public interface IInfoBarHost
	{
		Control AddInfoBar (InfoBarOptions options);
	}

 	public class InfoBarOptions
    	{
		public InfoBarItem[] Items { get; set; }

		public InfoBarOptions(string description)
                 {
                 }
	}
```

And we add `IInfoBarHost` to `Workbench` which would route into `DefaultWorkbench`’s `IInfoBarHost` implementation.

> VS bug [#678441](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/678441)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-7] Comment by **xamarinc**



Fixed in version 7.8.0.414 (master)



Pull Request #5878 merged by: Greg Munn  
 Author: mono  
 Commit: 61a45fbf6e817cb7c92218eafb9c1cbc8b6807e6 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-10] Comment by **xamarinc**



Fixed in version 7.7.0.1284 (release-7.7)



Pull Request #5903 merged by: Greg Munn  
 Author: mono  
 Commit: d79de452a0c79783a9ca085e43172a8ccbab6bac (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-30] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #5878 merged by: Greg Munn  
 Author: mono  
 Commit: 61a45fbf6e817cb7c92218eafb9c1cbc8b6807e6 (xamarin/monodevelop)
